src/modules/analytics/segment.ts

```diff
@@ -1,7 +1,16 @@
+import { isbot } from 'isbot'
+
 export interface AnalyticsWindow extends Window {
   analytics: SegmentAnalytics.AnalyticsJS
 }
 
 export function getAnalytics() {
+  //Check if user agent is a bot
+  const isBot = isbot(window.navigator.userAgent)
+
+  if (isBot) {
+    return
```

Comment:
Could you please change this?

```suggestion
    return undefined
```